src/components/Sidebar.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import { List, ListItem } from 'material-ui/List';
+import Divider from 'material-ui/Divider';
+import { Link } from 'react-router-dom';
+import './Sidebar.css';
+
+class Sidebar extends Component {
+  render() {
+    const books = this.props.books;
+    const score = this.props.score;
+    const currentScore = score.current;
+    const possibleScore = score.possible;
+
+    const scoreDisplay = `You've answered ${currentScore} of ${possibleScore} questions correctly. Time to dive in!`;
+
+    return (
+      <div className="sidebar">
+        <MuiThemeProvider>
+          <List style={{ width: '100%', margin: '0 auto' }}>
+            <ListItem
+              style={{
+                fontSize: '16px',
+                fontWeight: 'bold',
+                lineHeight: '16px',
+              }}
+              primaryText="Progress"
+            />
+            <ListItem
+              style={{ fontSize: '12px', lineHeight: '12px' }}
+              primaryText={scoreDisplay}
+            />
+          </List>
+
+          <Divider />
+
+          <List style={{ width: '100%', margin: '0 auto' }}>
+            <ListItem
+              style={{
+                fontSize: '16px',
+                fontWeight: 'bold',
+                lineHeight: '16px',
+              }}
+              primaryText="Books"
+            />
+            {books.map(book => (
+              <Link
+                key={book.id}
+                style={{ textDecoration: 'none' }}
+                to={book.url}
+              >
+                <ListItem
+                  style={{ fontSize: '12px', lineHeight: '12px' }}
+                  primaryText={`${book.title} (${
+                    score.books[book.id].current
+                  }/${score.books[book.id].possible})`}
```

Comment:
Might as well just use `book` instead of `books[book.id]`

Reply:
Oh wow thanks for pointing this out. Silly mistake. The first two changes you commented on above are a result of me doing this. Will fix!

Reply:
If that is the case then you can do something like this:

```javascript
 {books.map(book, index => (
```

There shouldn't be a need for `book.id` when you can use the index (in this case since the index is a unique identifier for the book)